app/src/main/java/sdp/moneyrun/game/Game.java

```diff
@@ -34,6 +34,11 @@
     //Aux variables
     private String id;
     private boolean hasBeenAdded;
+
+    public boolean isStarted() {
```

Comment:
A small comment saying that this variable is used to indicate that a game has been started could be useful for clarity

Reply:
done